src/inverseRobot.js

```diff
@@ -7,7 +7,33 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  // check if the robot is an object
+  if (typeof robot !== 'object' || robot === null) {
+    return null;
+  }
+
+  // "robot"'s keys become values and values become "robot"'s keys
+  const turnedRobot = {};
+
+  for (const key in robot) {
+    const INCOMING_ROBOT_VALUE = robot[key];
```

Comment:
it's better to rename these variables to lowercase, basically we use upper + snake keys for static variables that won't be changed

learn more about javascript naming convention - [link](https://www.robinwieruch.de/javascript-naming-conventions/)